src/pages/AddJob.jsx

```diff
@@ -39,9 +39,10 @@ export default function AddJob() {
       onSuccess: () => {
         navigate("/");
```

Comment:
could you help to add a toast message for success and error. and remove the todo. see login page for example

Reply:
Added